README.md

```diff
@@ -1,44 +1,44 @@
 # NL Design Skeleton Application
 
-The NL Design Skeleton application is design for rapid application testing and prototype development on the NL Design System. It does so by providing a basic skeleton application with full NL Design system functionality that an developer can easily extend view locally and deploy to an online environment for demonstration purposes. the main benefits of this are
+The NL Design Skeleton application is designed for rapid application testing and prototype development on the NL Design System. It provides a basic skeleton application with full NL Design system functionality that any developer can easily extend view locally and deploy to an online environment for demonstration purposes. The main benefit of this is:
 
-Development and (online) demonstration of prototypes without the need of an servers
-An ready out of the box basic  application that doesn't require configuration or setup and can be extended immediately.
+Development and (online) demonstration of prototypes without the need of a server.
+An out-of-the-box basic application that doesn't require configuration or setup and can be extended immediately.
```

Comment:
Dit is de `commonground-gateway-frontend` repository en niet die van de _Skeleton App_. Volgens mij staat deze documentatie nu daardoor niet op de juiste plek.

Deze repository bestaat namelijk (voor zover ik weet) niet om demonstrations e.d. te geven, maar om de Gateway in te regelen m.b.v. een GUI.